packages/cli/src/lib/codegen.ts

```diff
@@ -194,6 +194,20 @@ async function generateDefaultConfig(
   const defaultGlob = '*!(*.d).{ts,tsx,js,jsx}'; // No d.ts files
   const appDirRelative = relativePath(rootDirectory, appDirectory);
 
+  const caapiSchema = getSchema('customer-account');
+  const caapiProject = findGqlProject(caapiSchema, gqlConfig);
+
+  var customerAccountAPIConfig;
+  if (caapiProject?.documents) {
+    customerAccountAPIConfig = {
+      ['customer-accountapi.generated.d.ts']: {
+        preset,
+        schema: caapiSchema,
+        documents: caapiProject?.documents,
+      },
+    };
+  }
```

Comment:
Also, should we call this `customer-accountapi.generated.d.ts` or `customeraccountapi.generated.d.ts`? Kind of weird to have only 1 hyphen but I don't like it either without it, but more consistent with `storefrontapi.generated.d.ts` 🤦

Reply:
so I did thought about `customeraccountapi.generated.d.ts` and really dislike how hard it is to read it.
I ended up going with `customer-accountapi.generated.d.ts` because customer account are two word where storefront is actually one word so they are still consistent.